javascript/features/houses/utils/interior_list.js

```diff
@@ -301,6 +301,33 @@ const INTERIOR_LIST = [
             duration: 2500
         }
     }
```

Comment:
There needs to be a comma after this line.

Reply:
k